_tags

```diff
@@ -1,5 +1,5 @@
 true : bin_annot, safe_string
-not <ci/**>: warn_error(+1..49), warn(A-3-4-41-44)
+not <ci/**>: warn_error(+1..49-3), warn(A-4-41-44)
```

Comment:
@samoht: not sure I quite understand what this does. I think ideally we want:
- Show deprecation warnings when compiling normally, but don't abort.
- Make warnings fatal under CI.